ledger/src/shred.rs

```diff
@@ -123,8 +128,7 @@ pub const DATA_SHREDS_PER_FEC_BLOCK: usize = 32;
 static DATA_SHRED_BYTES_PER_BATCH_TYPICAL: OnceLock<u64> = OnceLock::new();
 pub fn get_data_shred_bytes_per_batch_typical() -> &'static u64 {
     DATA_SHRED_BYTES_PER_BATCH_TYPICAL.get_or_init(|| {
-        let proof_size = merkle::get_proof_size(DATA_SHREDS_PER_FEC_BLOCK * 2);
-        let capacity = ShredData::capacity(Some((proof_size, true, false)))
+        let capacity = ShredData::capacity(Some((PROOF_NUM_ENTRIES_TYPICAL, true, false)))
```

Comment:
Are we starting getting "coding batch" and "FEC block" terms referring to the same exact thing? Or am I missing some subtle detail here?

Reply:
yeah... we have multiple terms referring to the same concept:
* FEC set
* FEC block
* Erasure Batch
* Coding Batch

Reply:
I vote for FEC set/batch or Coding Batch. As long as we do not call them blocks =)